imagine/tests/test_templates.py

```diff
@@ -197,11 +175,20 @@ def test_pipeline_template():
                                        [0,0]*u.kpc],
                                   resolution=[30,1,1])
     # Field factories
-    TE_factory = FakeRandomTEFactory(grid=grid)
-    TE_factory.active_parameters = ['param']
-    B_factory = ConstantBFactory(grid=grid)
-    B_factory.active_parameters = ['Bx', 'By']
-
+    TE_factory = img_fields.FieldFactory(
+        field_class=FakeRandomTE,
+        grid=grid,
+        active_parameters=['param'],
+        priors={'param': img_priors.FlatPrior(xmin=0, xmax=10.)})
+
+    B_factory = img_fields.FieldFactory(
+      field_class=img_fields.ConstantMagneticField,
+      grid=grid,
+      active_parameters=['Bx', 'By'],
+      default_parameters={'Bz': 3.*muG},
```

Comment:
Just out of interest, why did you fix the Bz value here, while it was inferred previously?

Reply:
The idea was testing both the infrastructure for active and inactive parameters. Arbitrarily, I made Bx and By active, and Bz inactive.